.github/scripts/update-azure-specs-packages.ts

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env tsx
```

Comment:
formatting still broken
```
> @azure-tools/typespec-azure-monorepo@0.0.1 format:check /home/runner/work/typespec-azure/typespec-azure
> prettier . --check

Checking formatting...
[warn] .github/scripts/update-azure-specs-packages.ts
[warn] .github/scripts/validate-typespec-specs.ts
[warn] .github/workflows/external-integration.yml
panic: reflect: unimplemented: AssignableTo with interface
[warn] Code style issues found in 3 files. Run Prettier with --write to fix.
 ELIFECYCLE  Command failed with exit code 1.
```

make sure to run `pnpm format` at the root. DO NOT THINK you know what is correct formatting